src/uipath/_events/_sw_subscriber.py

```diff
@@ -0,0 +1,137 @@
+"""Event handlers for evaluation progress events."""
+
+import logging
+from typing import Any, Dict, List, Optional
+
+from uipath._cli._evals._models._sw_reporting import StudioWebProgressItem
+from uipath._cli._evals._progress_reporter import StudioWebProgressReporter
+from uipath._events._events import (
+    CreateEvalRunEvent,
+    CreateEvalSetRunEvent,
+    UpdateEvalRunEvent,
+    UpdateEvalSetRunEvent,
+)
+from uipath.eval.models import ScoreType
+
+logger = logging.getLogger(__name__)
+
+
+class StudioWebProgressSubscriber:
+    """Handler for Studio Web progress reporting events."""
+
+    def __init__(self):
+        """Initialize the handler."""
+        self.reporter = StudioWebProgressReporter()
+        self.eval_set_run_id: Optional[str] = None
```

Comment:
This subscriber should support multiple eval runtimes executions (use eval_context.execution_id to cache the eval runs)

Reply:
Basically all the events should also be linked to an eval runtime execution id